snot.py

```diff
@@ -183,6 +166,7 @@ def emit(self, record):
 class SlickAsSnotPlugin(nose.plugins.Plugin):
     name = "snot"
     score = 1800
+    testruns = dict()
```

Comment:
This dictionary stares slickqa objects but exists primarily to avoid duplicating testrun group creation. After creating a test run the slickqa object is stored to this dictionary, keyed with the testrun group `name`. During subsequent iterations through the test loop, this dictionary is checked to see if a key already exists with the testrun group name. If so, it re-uses the slickqa object to add the testresult to that testrun group.